### PR TITLE
Project: Clean includes

### PR DESCRIPTION
**Pull request by Hallot**
Remove some unnecessary includes.
Replace includes with forward declaration when possible.
#548

### CODE REVIEW

File: libopenage/console/draw.cpp [hunk 1]
```diff
@@ -8,18 +8,21 @@
...
 #include "../renderer/text.h"
+#include "../engine.h"
 
 namespace openage {
 namespace console {
 namespace draw {
 
 void to_opengl(Console *console) {
-	coord::camhud topleft = {
+    coord::camhud topleft = {
```
Comment (TheJJ) [inline libopenage/console/draw.cpp]: you changed indentation from tab to space here :P

**Comment by TheJJ** [comment]
Thanks! As you can see on the details link from the ci (best viewed with `curl`), you need to update the file copyright years in files you changed, and add yourself to `copying.md`.

Also, would be cool if you sorted the headers by name, especially the ones you add to existing blocks.

**Comment by Hallot** [comment]
Thanks, I'll squash once everything is good for you.

**Comment by TheJJ** [comment]
Fancy new github can squash directly :)

**Pull request merged**